### PR TITLE
Feature/nvme cli log page directory add log page names

### PR DESCRIPTION
**Pull request by lgdacunh**

### CODE REVIEW

File: nvme-print.c [hunk 1]
```diff
@@ -664,12 +664,22 @@ const char *nvme_log_to_string(__u8 lid)
 	case NVME_LOG_LID_ENDURANCE_GROUP:		return "Endurance Group Information";
 	case NVME_LOG_LID_PREDICTABLE_LAT_NVMSET:	return "Predictable Latency Per NVM Set";
 	case NVME_LOG_LID_PREDICTABLE_LAT_AGG:		return "Predictable Latency Event Aggregate";
+	case NVME_LOG_LID_MEDIA_UNIT_STATUS:
+		return "Media Unit Status";
+	case NVME_LOG_LID_SUPPORTED_CAP_CONFIG_LIST:
+		return "Supported Capacity Configuration List";
```
Comment (igaw) [inline nvme-print.c:670]: Although the lines get a bit long with the new entries, I'd rather stay consistent and keep every entry  one one line.

**Review by igaw — CHANGES_REQUESTED** [review]
A formatting nitpicking and I've noticed that shankara is listed as author, so the `Signed-off-by`  from shankara would be necessary.

Rest looks good.

File: nvme-print.c (hunk 1 above)
Comment (lgdacunh) [reply nvme-print.c:670]: We are working to get Shankar the proper GitHub user and keys. Soon we will have a signed commit with the requested changes.

**Comment by lgdacunh** [comment]
Shankar amended commits with his signature, and performed the requested code format changes.

**Review by igaw — APPROVED** [review]

**Comment by igaw** [comment]
Thanks!

**Pull request merged**